changelog/7.43.0/pr-2571.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: Json parser string length metrics are tagged by format
+  links:
+  - https://github.com/palantir/conjure-java-runtime/pull/2571
```

Comment:
Looks like something went sideways in a rebase/merge

Reply:
Ugh, the merge of origin/develop seems to have gone sideways, I may just put together a separate PR

Reply:
Moving to https://github.com/palantir/conjure-java-runtime/pull/2616